### ISSUE TITLE
Completely broken

### ISSUE THREAD

**Issue by longnguyen2004** [longnguyen2004/FacebookVideoOptimizer#5]
Yes, Facebook broke the thing, both File Size mode and Quality mode.

**Comment by SteveFour** [comment]
So for now it is impossible to keep newer 60+ fps videos in Facebook?

**Comment by longnguyen2004** [comment]
Yep, I tried changing codec, bitrate, etc. Seems like they're running every video through the same pipeline now. If anyone has a working video, please tell me.
